(2021.02.26) Stefan Verkoyen (sverkoyen.contractor@libertyglobal.com)
ONEM-18219 Support for Thunder on RDK3 based platforms like apollo

Remove unwanted calls to rfcapi, as to remove DEPENDS to rfc

Index: git/helpers/utils.cpp
===================================================================
--- git.orig/helpers/utils.cpp
+++ git/helpers/utils.cpp
@@ -165,14 +165,6 @@ void Utils::SecurityToken::getSecurityTo
 
     // Thunder Security is enabled by Default.
     bool thunderSecurityRFCEnabled = true;
-    RFC_ParamData_t param;
-    if (getRFCConfig("Device.DeviceInfo.X_RDKCENTRAL-COM_RFC.Feature.ThunderSecurity.Enable", param))
-    {
-        if (param.type == WDMP_BOOLEAN && (strncasecmp(param.value,"false",5) == 0))
-        {
-            thunderSecurityRFCEnabled = false;
-        }
-    }
     std::cout << "Thunder Security RFC enabled: " << thunderSecurityRFCEnabled << std::endl;
     if(!isThunderSecurityConfigured() || !thunderSecurityRFCEnabled)
     {
@@ -302,16 +294,6 @@ bool Utils::isPluginActivated(const char
     return pluginActivated;
 }
 
-bool Utils::getRFCConfig(char* paramName, RFC_ParamData_t& paramOutput)
-{
-    WDMP_STATUS wdmpStatus = getRFCParameter("RDKShell", paramName, &paramOutput);
-    if (wdmpStatus == WDMP_SUCCESS || wdmpStatus == WDMP_ERR_DEFAULT_VALUE)
-    {
-        return true;
-    }
-    return false;
-}
-
 std::string Utils::SecurityToken::m_sToken = "";
 bool Utils::SecurityToken::m_sThunderSecurityChecked = false;
 
